Nos/Views/Settings/SettingsView.swift

```diff
@@ -217,6 +219,32 @@ struct SettingsView: View {
                     .padding(.vertical, 15)
             }
             .listRowGradientBackground()
+            
+            #if STAGING || DEBUG
+            if isDeleteAccountEnabled.wrappedValue {
```

Comment:
```suggestion
            if featureFlags.isEnabled(.deleteAccount) {
```

With @Observable we can also reference the feature flags directly like this, avoiding the need for the `isDeleteAccountEnabled` Binding. I think I like this better? Because it's less code and it's more obvious at the call site where the source of truth is. But I could also see an argument that repeating this slightly more complex incantation across the view isn't dry... I'll leave it up to you. I'm approving and I'm good with merging whether you want to do a little refactor or not.